### PR TITLE
Add diagrams for Trinsic Connect flows

### PR DESCRIPTION
**Pull request by tmarkovski**

### CODE REVIEW

File: docs/platform/connect.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Trinsic Connect
+
+## What is Trinsic Connect?
+
```
Comment (janpieterz) [inline docs/platform/connect.md:4]: We should add something here. My suggestion below (but that's just rough drafting)

Comment (janpieterz) [inline docs/platform/connect.md:4]:
```suggestion

*Overview*

Unleash the power of reusable ID verification. Say goodbye to repeated verification hassles. Verify once, use endlessly.

*Features*

1. One-and-Done Verification: Send users through our thorough document verification process just once.
1. Unlimited Reusability: Once verified, the user's credentials are ready to be reused anywhere, anytime.
1. Seamless Integration: Compatible with both web and mobile apps. Get up and running in minutes.
1. Brand-Friendly: Tailor the look and feel to match your brand's vibe. Keep the experience consistent.

```

File: docs/platform/connect.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    T->>U: Ask consent to share reusable VC
+    deactivate T
+    U->>RP: Consent granted.<br/>Notify process completed
+    RP->>T: Get results for Session ID
+    T-->>RP: Return proof of VC
+    deactivate U
+
+```
+
+## How to integrate?
```
Comment (janpieterz) [inline docs/platform/connect.md:93]:
```suggestion
## How to integrate?

1. Contact the Trinsic team to get access to the configuration portal.
1. Access the [Trinsic Studio](https://idv.connect.trinsic.cloud) to configure your Connect integration and retrieve your auth token.
1. See our [Identity Verification Examples](https://docs.trinsic.id/examples/#identity-document-verfications) for easy ways to get started
```

**Review by janpieterz — APPROVED** [review]

File: docs/platform/connect.md (hunk 1 above)
Comment (tmarkovski) [reply docs/platform/connect.md:4]: This is just a handoff to @Hmac512 who will be working on the content.

**Comment by tmarkovski** [comment]
Obsolete

**Pull request closed without merging**